jobs/generated/e2e-gce-ubuntustable1-k8snew-default.env

```diff
@@ -0,0 +1,24 @@
+# Auto-generated; DO NOT MODIFY.
+
+# The common configurations.
+
+# The cloud provider configurations.
+KUBERNETES_PROVIDER=gce
+E2E_MIN_STARTUP_PODS=8
+KUBE_GCE_ZONE=us-central1-f
+CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
+
+# The image configurations.
+KUBE_NODE_OS_DISTRIBUTION=ubuntu
+KUBE_GCE_NODE_PROJECT=ubuntu-os-gke-cloud
+KUBE_GCE_NODE_IMAGE=ubuntu-gke-1604-xenial-v20170420-1
+
+# The k8s version configurations.
+
+# The test suite configurations.
+GINKGO_PARALLEL=y
+GINKGO_PARALLEL_NODES=30
+GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]
+
+# The job configurations.
+PROJECT=ubuntu-image-validation
```

Comment:
you'll need to add something to https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap_test.py#L2000-L2023